pkg/controllers/provisioningv2/managedchart/managedchart.go

```diff
@@ -92,15 +92,15 @@ func (h *handler) OnChange(mcc *v3.ManagedChart, status v3.ManagedChartStatus) (
 
 	bundle := &v1alpha1.Bundle{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      name.SafeConcatName("mcc", mcc.Name),
+			Name:      name.Limit(name.SafeConcatName("mcc", mcc.Name), 53),
```

Comment:
Also, do we have to shorten the Name itself (vs. ReleaseName). The change that introduced the limit doesn't seem to apply to Name (https://github.com/rancher/fleet/pull/964/files#diff-fe6024d9cdd994ea2d530051f203572ee902f5af2936f095b2bfd4f81f4aca26) but I may be missing something.

Reply:
I believe the release name is derived from the name in cases where one is not explicitly provided, which is why I shorted the name too. This was the same change made for the SUC bundle and wanted to cover both cases